modules/shortcodes/vr.php

```diff
@@ -68,7 +68,12 @@ function jetpack_vr_viewer_shortcode( $atts ) {
 		return jetpack_vr_viewer_get_html( $url_params );
 	}
 
-	return '[vr] shortcode requires a data source to be given';
+	// add check for user
+	if ( current_user_can('editor') || current_user_can('administrator') ) {
```

Comment:
Another example is a new role `basic_editor` created with Members plugin with only the `edit_posts` capability. It's not `editor` nor `administrator` but it can still edit the posts.